README.md

```diff
@@ -25,83 +36,30 @@ cd servicebox-app
 ./mvnw compile quarkus:dev
```

Comment:
the script `mvnw` does not exist. 
@cmoulliard opened an issue related to this. @aureamunoz can you also fix it as part of this ticket?

Reply:
ok

Reply:
done